coa-helm/templates/csi/csi-blob-controller.yaml

```diff
@@ -0,0 +1,133 @@
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: csi-blob-controller
+  namespace: kube-system
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: csi-blob-controller
+  template:
+    metadata:
+      labels:
+        app: csi-blob-controller
```

Comment:
I see the word "blob", and I see that "CSI" (in the context of kubernetes) can mean "container storage interface". Is it easy to explain what this container is doing? (I assume it's related to the blobfuse container above)

Reply:
Yes, this is setting up the mount of the Blob containers so that Cromwell can access them as a local filesystem. I'm not deeply familiar with the details of how the blobfuse situation is set up. It won't be our long-term solution, so I haven't dug in too deeply - I wouldn't work too hard to understand anything in this `csi` subdirectory (something I should have told you at the beginning...).

Reply:
thanks! I wont :)